### PR TITLE
ISPN-14983 Ensure ClusterRoleMapper and ClusterPermissionMapper are i…

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-14983

### CODE REVIEW

**Comment by tristantarrant** [comment]
Failures are unrelated

**Comment by ryanemerson** [comment]
Thanks @tristantarrant

File: core/src/main/java/org/infinispan/factories/GlobalComponentRegistry.java [hunk 1]
```diff
@@ -279,8 +279,7 @@ protected void preStart() {
       }
    }
 
-   @Override
-   protected void postStart() {
```
Comment (fax4ever) [inline core/src/main/java/org/infinispan/factories/GlobalComponentRegistry.java:283]: Why removing it?

File: core/src/main/java/org/infinispan/globalstate/GlobalConfigurationManager.java [hunk 2]
```diff
@@ -21,6 +21,8 @@
 public interface GlobalConfigurationManager {
    String CONFIG_STATE_CACHE_NAME = "org.infinispan.CONFIG";
 
+   default void postStart() {}
+
```
Comment (fax4ever) [inline core/src/main/java/org/infinispan/globalstate/GlobalConfigurationManager.java:25]: Is it used?

**Review by fax4ever — COMMENTED** [review]
It looks good to me. Just a doubt here

File: core/src/main/java/org/infinispan/factories/GlobalComponentRegistry.java (hunk 1 above)
Comment (tristantarrant) [reply core/src/main/java/org/infinispan/factories/GlobalComponentRegistry.java:283]: It was deprecated a long while ago.

File: core/src/main/java/org/infinispan/globalstate/GlobalConfigurationManager.java (hunk 2 above)
Comment (tristantarrant) [reply core/src/main/java/org/infinispan/globalstate/GlobalConfigurationManager.java:25]: No

**Comment by fax4ever** [comment]
Sorry my review came too late :P

**Pull request merged**